README.md

```diff
@@ -4,7 +4,7 @@
 
 Rules used by the Administrative Steering Committee (ASC) for the PMIx Standard.
 
- * Current version of the [The PMIx Standard Governance Rules](https://pmix.org/wp-content/uploads/2020/01/pmix_governance-2020-01-27.pdf)
+ * Current version of the [The PMIx Standard Governance Rules](https://github.com/pmix/governance/releases/tag/v1.2)
```

Comment:
```suggestion
 * Current version of the [The PMIx Standard Governance Rules](https://github.com/pmix/governance/releases/latest)
```

Reply:
Good idea. I didn't know about the `latest` tag.